examples/tox21/tox21_tensorgraph_graph_conv.py

```diff
@@ -18,87 +22,94 @@
 import deepchem as dc
 from tox21_datasets import load_tox21
 
+model_dir = "/tmp/graph_conv"
 
-def graph_conv_model(batch_size, num_tasks):
-  model = GraphConvTensorGraph(batch_size=batch_size, use_queue=False)
-  atom_features = Input(shape=(None, 75))
-  model.add_feature(atom_features)
-
-  degree_slice = Input(shape=(None, 2), dtype=tf.int32)
-  model.add_feature(degree_slice)
 
-  membership = Input(shape=(None,), dtype=tf.int32)
-  model.add_feature(membership)
+def graph_conv_model(batch_size, tasks):
+  model = TensorGraph(
+      model_dir=model_dir, batch_size=batch_size, use_queue=False)
+  atom_features = Feature(shape=(None, 75), name="ATOMFEATURES")
```

Comment:
Let's remove the key-word args and the explicit names here to make the example maximally simple for newbies.